<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/2.png</normaloff>:/picture/2.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_point">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/dot.png</normaloff>:/picture/dot.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/1.png</normaloff>:/picture/1.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/0.png</normaloff>:/picture/0.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_enter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/enter.png</normaloff>:/picture/enter.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/3.png</normaloff>:/picture/3.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_back">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/back.png</normaloff>:/picture/back.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/5.png</normaloff>:/picture/5.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/4.png</normaloff>:/picture/4.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="display">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_clear">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/clear.png</normaloff>:/picture/clear.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/7.png</normaloff>:/picture/7.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/6.png</normaloff>:/picture/6.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_sign">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-/+</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/sign.png</normaloff>:/picture/sign.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/8.png</normaloff>:/picture/8.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/picture/9.png</normaloff>:/picture/9.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
